metrics-jetty12-ee10/pom.xml

```
@@ -0,0 +1,139 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.dropwizard.metrics</groupId>
+        <artifactId>metrics-parent</artifactId>
+        <version>4.2.20-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>metrics-jetty12-ee10</artifactId>
+    <name>Metrics Integration for Jetty 12.x and higher with Jakarta EE 10</name>
+    <packaging>bundle</packaging>
+    <description>
+        A set of extensions for Jetty 12.x and higher which provide instrumentation of thread pools, connector
+        metrics, and application latency and utilization. This module uses the Servlet API from Jakarta EE 10.
+    </description>
+
+    <properties>
+        <javaModuleName>io.dropwizard.metrics.jetty12.ee10</javaModuleName>
+
+        <maven.compiler.source>17</maven.compiler.source>
+        <maven.compiler.target>17</maven.compiler.target>
```

```suggestion
        <maven.compiler.release>17</maven.compiler.release>
```
See also: https://maven.apache.org/plugins/maven-compiler-plugin/examples/set-compiler-release.html